Cakes-Co/index.html

```diff
@@ -1,16 +1,183 @@
+            <a href="#" class="cakes">CAKES</a>
+            <a href="#" class="order-in">ORDER IN</a>
+            <a href="#" class="lessons">LESSONS</a>
+            <a href="#" class="about">ABOUT</a>
+        </div>
+      </nav>
+
+      <!-- MAIN SECTION -->
 
-</html>
\ No newline at end of file
+      <main>
+        <div></div>
+        <div>
+          <h1>Welcome</h1>
+          <p>
+            Indulge in the sweet world of delectable delights with us! We're
+            thrilled to have you join our cake-loving community. Whether you're
+            a baking enthusiast, a cake connoisseur, or just someone with a
+            sweet tooth, you've come to the right place. At [Your Cake Website],
+            we're passionate about all things cake, from classic recipes to
+            innovative creations. Here, you'll find mouthwatering recipes,
+            expert tips, and a treasure trove of cake inspiration. Get ready to
+            embark on a delicious journey where sugar and creativity collide!
+            Don't forget to explore our gallery of stunning cake designs and
+            flavors. Looking for that perfect birthday cake, a show-stopping
+            wedding centerpiece, or just a tasty treat for yourself? We've got
+            you covered.
+          </p>
+        </div>
+      </main>
+
+      <!-- CONTENT SECtION -->
+
+      <content>
+        <div>
+          <img
+            src="https://images.unsplash.com/photo-1614707267537-b85aaf00c4b7?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=3072&q=80"
+            width="100%"
+            height="100%"
+          />
+        </div>
+        <div>
+          <img
+            src="https://images.unsplash.com/photo-1576618148400-f54bed99fcfd?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2334&q=80"
+            width="100%"
+            height="100%"
+          />
+        </div>
+        <div>
+          <img
+            src="https://images.unsplash.com/photo-1599785209796-786432b228bc?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=3086&q=80"
+            width="100%"
+            height="100%"
+          />
+        </div>
+        <div>
+          <img
+            src="https://images.unsplash.com/photo-1615837136007-701de6015cfb?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTh8fGN1cGNha2V8ZW58MHx8MHx8fDA%3D&auto=format&fit=crop&w=800&q=60"
+            width="100%"
+            height="100%"
+          />
+        </div>
+      </content>
+
+      <!-- FOOTER SECTION -->
+
+      <footer>
+        <div>
+          <a href="#" class="footer-links">Contact</a>
+          <a href="#" class="footer-links">Custom Catering</a>
+          <a href="#" class="footer-links">Email Signup</a>
+        </div>
+        <div class="socials-svgs">
+          <a href="#">
+            <svg
```

Comment:
Look into https://svgomg.net/

Did you know you can link an svg as the arc for an img tag? Why might someone do this?

Reply:
Could you explain this in more detail please?